pkgs/tools/package-management/nix/default.nix

```diff
@@ -242,6 +253,14 @@ in lib.makeExtensible (self: ({
     hash = "sha256-EtL6M0H5+0mFbFh+teVjm+0B+xmHoKwtBvigS5NMWoo=";
   };
 
+  nix_2_20 = common {
+    version = "2.20.4";
+    hash = "sha256-Sv5VFPF5BAXkMWgekh0iH1SeqTF8VcCiW5nR6/AATrI=";
+    patches = [
+      ./2_20/CVE-2024-27297.patch
```

Comment:
Should be `./patches/2_20`, right? DOesn't evaluate for me at least.

Reply:
or update to 2.20.5

Reply:
yes... sorry

Reply:
feel free to update to 2.20.5 otherwise

Reply:
Dropped the patch by updating to 2.20.5